addOns/tlsdebug/src/main/java/org/zaproxy/zap/extension/tlsdebug/HttpsCallerProcess.java

```diff
@@ -55,7 +58,7 @@ public static void main(String[] args) {
             }
             System.out.println("--- END of call -------------------------------------------------");
         } catch (MalformedURLException e) {
-            e.printStackTrace();
```

Comment:
This is actually correct, this is a process spawn by the add-on which consumes its output.